App/Components/Header.js

```diff
@@ -6,61 +6,73 @@ import React, {
   View
 } from 'react-native';
 
+import appColors from './_appColors';
+
 class Header extends Component {
+
   goBack() {
     if(this.props.navigator){
       this.props.navigator.pop();
     }
   }
+
   render() {
     return (
-      <View style={style.container}>
-        <TouchableHighlight underlayColor={"'rgba(255, 255, 255, 0.05)'"} onPress={this.goBack.bind(this)}>
-          <View style={style.backButton}>
-            <Text style={style.text}>
+      <View style={styles.container}>
+        <TouchableHighlight
+          underlayColor={"'rgba(255, 255, 255, 0.05)'"}
+          onPress={this.goBack.bind(this)}>
+          <View style={styles.backButton}>
+            <Text style={styles.text}>
               &lsaquo;
             </Text>
           </View>
         </TouchableHighlight>
-        <View style={style.titleContainer}>
-          <Text style={style.title}>
+        <View style={styles.titleContainer}>
+          <Text style={styles.title}>
             {this.props.value}
           </Text>
         </View>
       </View>
     );
   }
+
 }
 
-const style = StyleSheet.create({
+const styles = StyleSheet.create({
+
   container: {
     marginTop: 15,
     flexDirection: 'row',
     height: 60,
     borderBottomWidth: 1,
-    borderColor: '#0785ab',
+    borderColor: appColors.borderColorHeader,
     flex: 1
   },
+
   backButton: {
-    // flex: 1, 
     alignItems: 'flex-start',
     marginLeft: 5
   },
+
   text: {
-    color: '#fff',
+    color: appColors.defaultTextColor,
     fontSize: 48
   },
-  title: {
-    fontSize: 24,
-    color: '#fff'  
-  },
+
   titleContainer: {
     marginTop: 16,
     marginBottom: 15,
     flex: 1,
     alignItems: 'center',
 
+  },
+
+  title: {
+    fontSize: 24,
```

Comment:
maybe we can abstract this title fontSize into a value too? since that value will probably never change.